packages/terra-application-utility/src/utility/_UtilityMenu.module.scss

```
@@ -19,81 +19,102 @@
   }
 
   .menu-utility-menu-content-container {
-    margin-bottom: var(--terra-application-menu-utility-header-margin-bottom, 0.7143rem);
+    margin-bottom: var(--terra-application-utility-menu-type-menu-header-margin-bottom, 0.7143rem);
   }
 
   .utility-menu-left-content-container {
     align-items: center;
     display: flex;
     justify-content: flex-start;
+    overflow: hidden;
   }
 
   .header-utility-menu-initial-page-header-text {
-    margin-left: var(--terra-application-header-utility-menu-initial-page-header-text-margin-left, 0.7143rem);
+    margin-left: var(--terra-application-utility-header-type-menu-initial-page-header-text-margin-left, 0.7143rem);
+    overflow: hidden;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
   }
 
   .menu-utility-menu-initial-page-header-text {
-    margin-left: var(--terra-application-menu-utility-menu-initial-page-header-text-margin-left, 0.7143rem);
+    margin-left: var(--terra-application-utility-menu-type-menu-initial-page-header-text-margin-left, 0.7143rem);
+    overflow: hidden;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
   }
 
   .header-utility-menu-noninitial-page-header-text {
-    margin-left: var(--terra-application-header-utility-menu-noninitial-page-header-text-margin-left, 0);
+    margin-left: var(--terra-application-utility-header-type-menu-noninitial-page-header-text-margin-left, 0);
+    overflow: hidden;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
   }
 
   .menu-utility-menu-noninitial-page-header-text {
-    margin-left: var(--terra-application-menu-utility-menu-noninitial-page-header-text-margin-left, 0);
+    margin-left: var(--terra-application-utility-menu-type-menu-noninitial-page-header-text-margin-left, 0);
+    overflow: hidden;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
   }
 
   .utility-menu-icon-left {
     flex-shrink: 0;
   }
 
   .header-utility-menu-icon-left {
-    width: var(--terra-application-header-utility-menu-chevron-width, 0.7143rem);
+    width: var(--terra-application-utility-header-type-menu-back-width, 0.7143rem);
   }
 
   .menu-utility-menu-icon-left {
-    width: var(--terra-application-menu-utility-menu-chevron-width, 0.7143rem);
+    width: var(--terra-application-utility-menu-type-menu-back-width, 0.7143rem);
   }
 
   .utility-menu-right-content-container {
     display: flex;
     justify-content: flex-end;
   }
 
+  .header-utility-menu-button-close {
+    display: var(--terra-application-utility-header-type-menu-button-close-display);
+  }
+
   .utility-menu-icon-close {
     flex-shrink: 0;
   }
 
   .header-utility-menu-icon-close {
-    width: var(--terra-application-header-utility-menu-page-chevron-width, 1rem);
+    display: var(--terra-application-utility-header-type-menu-page-display);
+    width: var(--terra-application-utility-header-type-menu-page-close-width, 1rem);
```

nit: This may be more clear if we add `icon` to the name
```suggestion
    width: var(--terra-application-utility-header-type-menu-page-close-icon-width, 1rem);
```

---

Fixed in https://github.com/cerner/terra-framework/pull/439/commits/efbb9e2eb8d2c426fa2993fa3bb0d20efb23c927